modules/EvseManager/evse/evse_managerImpl.cpp

```diff
@@ -127,23 +127,20 @@ void evse_managerImpl::ready() {
             se.session_started = session_started;
         }
 
-        if (e == Charger::EvseEvent::SessionFinished) {
```

Comment:
this and following should be else if or use switch case

Reply:
ok